config.mk

```diff
@@ -49,16 +49,16 @@ arm_gcc_tarball_type=xz
 # recommended. The ISL dependency isn't mandatory; if you don't want it, you may 
 # just comment the version numbers (i.e. 'sh_isl_ver' and 'arm_isl_ver') to
 # disable the ISL library.
-#use_custom_dependencies=1
+use_custom_dependencies=1
```

Comment:
Why using custom dependencies for newer GCC like 10? This parameter was used in the past for GCC 3 which doesn't provides `download_prerequisites` script.